app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileViewModel.kt

```diff
@@ -44,12 +44,12 @@ class UserProfileViewModel @Inject constructor(
             getSelf().collect {
                 userProfileState = SelfUserProfileState(
                     status = UserStatus.AVAILABLE,
-                    fullName = it.name!!,
-                    userName = it.handle!!,
+                    fullName = it.name.orEmpty(),
```

Comment:
the user `fullname` should not be nullable at the first place, since a user can't register without a name and if it's null that means there is an error somewhere